### DISCUSSION TITLE
how to do chrome://omnibox WebView ??

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1335] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
??

**Comment by Charlieis1nsan3** [comment]
idk that one But I do know this one, 
1. go to chrome://chrome-signin
2. type in google@d11.org
3. click sign-in options
4. click github
5. scroll down and click Privacy
6. it should take to the docs. scroll to the very bottom until you find  "Ask the Github community"
7. click that
8. once you are redirected type "Google" in the search bar
9. and once you are done in the right you will see Google LLC is an American multinational technology company that specializes in Internet-related services and products.
[google.com](https://www.google.com/)
[Wikipedia](https://en.wikipedia.org/wiki/Google)
[google](https://github.com/google)
[View topic](https://github.com/topics/google)

finally click the Google.com

**Comment by ghost** [comment]
chrome://chrome-signin
 blocker but thanks

**Comment by buddyted** [comment]
that doesn't work for me is there a way to get around this
![Screenshot 2024-12-03 11 06 01 AM](https://github.com/user-attachments/assets/de269ae1-0533-4379-83ba-1d277ee8ac7c)

**Reply by no-name-6655** [reply]
No

**Comment by Charlieis1nsan3** [comment]
`<!DOCTYPE HTML>
<p> fortnite balls all in your face </p
 <button draggable=false>Click for free V-Bucks </button>`

**Reply by ghost** [reply]
bet!

**Comment by S-PScripts** [comment]
chrome://omnibox  has a webview?

**Reply by ghost** [reply]
I saw it in the Tn Discord IN WebView kaigs 
![image](https://github.com/user-attachments/assets/42f5900d-f074-4ebc-9be6-547c41a44346)

**Reply by schoolexploitkid** [reply, marked as answer]
the omnibox is the url bar, this is just a joke

**Reply by S-PScripts** [reply]
> I saw it in the Tn Discord IN WebView kaigs ![image](https://private-user-images.githubusercontent.com/135744675/392158216-42f5900d-f074-4ebc-9be6-547c41a44346.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMzMDU3NTUsIm5iZiI6MTczMzMwNTQ1NSwicGF0aCI6Ii8xMzU3NDQ2NzUvMzkyMTU4MjE2LTQyZjU5MDBkLWYwNzQtNGViYy05YmU2LTU0N2M0MWE0NDM0Ni5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwNFQwOTQ0MTVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMWFhOTYxYWFkOTEwYWJkMTEyNWVlNDcwYWI4ZTcyM2JiMjAwOTU0MWI0NTUyYjYzM2ViODE2Mzk1YzEyZTE4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.LArR7yoy9Bb7x3OazLIGQEAYm0ewsFmYrj-bdSHg7KI)

bruh
